server/apps/immich/src/api-v1/oauth/oauth.service.ts

```diff
@@ -75,7 +57,8 @@ export class OAuthService {
 
     // register new user
     if (!user) {
-      if (!this.autoRegister) {
+      const { autoRegister } = await this.immichConfigService.getOAuthConfig();
+      if (!autoRegister) {
         this.logger.warn(
           `Unable to register ${profile.email}. To enable auto registering, set OAUTH_AUTO_REGISTER=true.`,
```

Comment:
This error message still refers to the old env vars. May want to check other log lines as well.

Reply:
Good catch!